test/cases/~hmr-resilience/expected/check.js

```diff
@@ -0,0 +1,19 @@
+(window["webpackJsonp"] = window["webpackJsonp"] || []).push([["check"],{
+
+/***/ "./index.css":
+/***/ (function(module, exports, __webpack_require__) {
+
+// extracted by mini-css-extract-plugin
+    if(true) {
+      // 1479427200000
+      var cssReload = __webpack_require__("../../../src/hmr/hotModuleReplacement.js")(module.i, {"hmr":true,"locals":false});
+      module.hot.dispose(cssReload);
+      module.hot.accept(function(){});
```

Comment:
Why?

Reply:
A little more context please:
why _what_ ?

Reply:
Why we need this `module.hot.accept(function(){});`? here, please add comment to source doe, because this line is misleading for other developers

Reply:
Right. Will add a small description in the loader.js where this particular line of code is emitted.

Reply:
Done